From 893c4a4089f796d1282b3ebde3d327db00e07157 Mon Sep 17 00:00:00 2001
From: Anders Dellien <anders.dellien@arm.com>
Date: Tue, 8 Jun 2021 09:35:56 +0100
Subject: [PATCH 1/2] board: armltd: Remove bootargs from Total Compute
 configuration

This information will be maintained in the device tree instead.

Signed-off-by: Anders Dellien <anders.dellien@arm.com>
Change-Id: I279399d4a0ea1a3330de5b58e8c9af78e48ba04c

Upstream-Status: Pending [https://lists.denx.de/pipermail/u-boot/2021-June/452156.html]
---
 configs/total_compute_defconfig | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/configs/total_compute_defconfig b/configs/total_compute_defconfig
index e36d701ff4..954023fae5 100644
--- a/configs/total_compute_defconfig
+++ b/configs/total_compute_defconfig
@@ -11,8 +11,7 @@ CONFIG_FIT=y
 CONFIG_FIT_SIGNATURE=y
 CONFIG_LEGACY_IMAGE_FORMAT=y
 CONFIG_BOOTDELAY=5
-CONFIG_USE_BOOTARGS=y
-CONFIG_BOOTARGS="console=ttyAMA0 debug user_debug=31 earlycon=pl011,0x7ff80000 loglevel=9 androidboot.hardware=total_compute video=640x480-32@60 androidboot.boot_devices=1c050000.mmci ip=dhcp androidboot.selinux=permissive"
+# CONFIG_USE_BOOTARGS is not set
 # CONFIG_USE_BOOTCOMMAND is not set
 # CONFIG_DISPLAY_CPUINFO is not set
 # CONFIG_DISPLAY_BOARDINFO is not set
-- 
2.17.1

